airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeDestination.java

```diff
@@ -53,4 +53,18 @@ public SerializedAirbyteMessageConsumer getSerializedMessageConsumer(final JsonN
 
   }
 
+  private static boolean useAsyncSnowflake(final JsonNode config) {
+    // this is how we toggle async snowflake on.
+    final Set<String> stagingLoadingMethods = Collections.emptySet();
```

Comment:
@davinchia This was toggled off when I started looking at it, should we be toggling this on? or are we just trying to get this merged?